server/opts.go

```diff
@@ -1051,6 +1053,10 @@ func parseJetStream(v interface{}, opts *Options, errors *[]error, warnings *[]e
 		switch strings.ToLower(mk) {
 		case "store_dir", "storedir":
 			opts.StoreDir = mv.(string)
+		case "maxmemory", "max_memory", "max_mem":
+			opts.JetStreamMaxMemory = mv.(int64)
+		case "maxstore", "max_store":
+			opts.JetStreamMaxStore = mv.(int64)
```

Comment:
I think perhaps we want to say max file store rather than just max store - so I imagine we can later allow some form of cloud object storage for long term archival and this option would be awkward then?

Reply:
Agree re: max_file_store.

Reply:
OK also made `max_memory_store` - some one on the channel got confused by this he thought it would also limit all the go routines etc to this size memory foot print, so this is clearer.

Reply:
We do need the max_memory IMO, but that is for future work.